urg3d_node2:
  ros__parameters:
    ip_address: '192.168.0.10'
    ip_port: 10940
    frame_id : 'hokuyo3d'
    range_min: 0.0
    interlace_h: 3
    interlace_v: 3
    output_cycle: 'line'
    publish_intensity: false
    publish_auxiliary: false
    calibrate_time : false
    synchronize_time : false
    error_limit : 4
    error_reset_period : 5.0
    diagnostics_tolerance : 0.05
    diagnostics_window_time : 5.0
